docs/src/architecture.md

```diff
@@ -159,6 +172,84 @@ Kubernetes cluster.
     coordinate PostgreSQL active/passive topologies across different Kubernetes
     clusters through a higher-level operator or management tool.
 
+### Reserving Nodes for PostgreSQL Workloads
+
+Whether you're operating in a multi-availability zone environment or, more
+critically, within a single availability zone, in production we strongly
+recommend isolating PostgreSQL workloads by dedicating specific worker nodes
+exclusively to `postgres`. This approach ensures optimal performance and
+resource allocation for your database operations.
+
+!!! Hint
+    As a general rule of thumb, deploy PostgreSQL nodes in multiples of
```

Comment:
"PostgreSQL nodes" --- I had to do a double-read here.
We need to be very clear we're talking about Kubernetes nodes dedicated to Postgres.
I may add a bit more verbiage here.

Reply:
That whole section is about worker nodes. It should be clear. But feel free to change it. Maybe "deploy Kubernetes worker nodes for PostgreSQL"

Reply:
Can you please check now?